### PR TITLE
Added cluster creation to slurmdb install

### PR DESCRIPTION
**Pull request by Slugger70**
If you install slurmdb and you want to use sacct to look at jobs, you need to create a "cluster" in the database with the same name as the one in slurm.conf.

This PR adds ansible code to do this.

### CODE REVIEW

**Comment by refual** [comment]
As of Slurm 20.02, this is no longer needed as `slurmctld` will add the cluster to the database on startup if it does not exist.

File: tasks/slurmdbd_cluster.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  become: yes
+  become_user: root
+
+- name: set cluster_check_boolean
+  set_fact:
+      __cluster_not_setup: false
+  when: cluster_check.stdout == "cluster"
+
+- name: Create the slurmdbd cluster
+  command: sacctmgr -i -n add cluster {{ __slurm_cluster_name }}
```
Comment (hexylena) [inline tasks/slurmdbd_cluster.yml:15]: this needs to be quoted or?

Comment (natefoo) [reply tasks/slurmdbd_cluster.yml:15]: Not for the command module I believe since it doesn't run through a shell, and spaces are invalid in cluster names.

**Comment by natefoo** [comment]
I successfully used this library in a recent one-off project, which is a little nicer to work with than a bunch of shell commands: https://github.com/dylex/ansible-hpc/blob/master/slurm.py

**Pull request merged**